external-deps/spyder-kernels/.gitrepo

```diff
@@ -7,6 +7,6 @@
 	remote = https://github.com/spyder-ide/spyder-kernels.git
 	branch = 1.x
 	commit = 1a9d44950edb462188f52cec51c975b80f5f0f6f
-	parent = adbc02315db0e49bbaa2cd6dae94526264398fc1
+	parent = ff7e4d25db6a366b51521870d76274a5e63dd0e8
```

Comment:
Please revert this change. It's not necessary because the latest version in the spyder-kernels subrepo contains the changes in spyder-ide/spyder-kernels#231.